Neos.Neos/Tests/Behavior/Features/Fusion/FlowQuery.feature

```diff
@@ -395,3 +395,49 @@ Feature: Tests for the "Neos.ContentRepository" Flow Query methods.
     removeNode: a
     nothingToRemove: a1a4,a1a4,a1a4
     """
+
+  Scenario: Node accessors (final Node access operations)
+    When the Fusion context node is "a1"
+    When I execute the following Fusion code:
+    """fusion
+    test = Neos.Fusion:DataStructure {
+      property = ${q(node).property('title')}
+      identifier = ${q(node).id()}
+      label = ${q(node).label()}
+      nodeTypeName = ${q(node).nodeTypeName()}
+      @process.render = ${Json.stringify(value, ['JSON_PRETTY_PRINT'])}
+    }
+    """
+    Then I expect the following Fusion rendering result:
+    """
+    {
+        "property": "Node a1",
+        "identifier": "a1",
+        "label": "Neos.Neos:Test.DocumentType1",
+        "nodeTypeName": "Neos.Neos:Test.DocumentType1"
+    }
+    """
+    # changing the node type config will invoke the Neos.Neos:FallbackNode handling
+    When I change the node types in content repository "default" to:
+    """yaml
+    'Neos.Neos:FallbackNode': {}
+    """
+    When I execute the following Fusion code:
+    """fusion
+    test = Neos.Fusion:DataStructure {
+      property = ${q(node).property('title')}
+      identifier = ${q(node).id()}
+      label = ${q(node).label()}
+      nodeTypeName = ${q(node).nodeTypeName()}
+      @process.render = ${Json.stringify(value, ['JSON_PRETTY_PRINT'])}
+    }
+    """
+    Then I expect the following Fusion rendering result:
+    """
+    {
+        "property": "Node a1",
+        "identifier": "a1",
+        "label": "Neos.Neos:Test.DocumentType1",
+        "nodeTypeName": "Neos.Neos:FallbackNode"
```

Comment:
or `${q(node).nodeTypeName()}` could always return the _actual_ nodeTypeName ... though there was no api for this in Neos 8.3.

... or there could be another operation returning the `FallbackNode` if required or an eel helper or an parameter? `${q(node).nodeTypeName('NODE_TYPE_USE_FALLBACK')}`

Reply:
at the moment we use this:

```
# this eel helper also handles the Neos.Neos:FallbackNode node type mechanism
type = ${Neos.Node.getNodeType(node).name.value}
```

but that is bad as it loads the node type and is not lazy

Reply:
Right, mmm, but for _our_ case we could do wiht something more "hidden" compared to an operation?

Reply:
sure sure that can be `type = ${Neos.Node.getNodeTypeNameWithFallbackHandling()}`

Reply:
added

```
type = ${Neos.Node.isNodeTypeExistent(node) ? q(node).nodeTypeName() : 'Neos.Neos:FallbackNode'}
```

as this will make configuring `FallbackNode` at some point possible ;)